### DISCUSSION TITLE
How to visualize segmented pointcloud with legend (on Python)

### DISCUSSION THREAD

**Discussion by evbernardes** [isl-org/Open3D#6337] [Q&A]
I have a pointcloud segmented into a number of different labels (let's say, 3).

Is it possible to  visualize the pointcloud with some kind of `legend`, as is done with `matplotlib.pyplot`, so I can easily see which groups correspond to each exact label?

**Comment by ssheorey** [comment]
Hi @evbernardes you can use the 3DML visualizer for this.

![](https://github.com/isl-org/Open3D-ML/blob/master/docs/images/getting_started_ml_visualizer.gif?raw=true)

Here's documentation for using it with your own data:

https://github.com/isl-org/Open3D-ML/blob/master/docs/howtos.md#visualize-custom-data
